src/NodaTime/Instant.cs

```diff
@@ -655,6 +655,28 @@ public static Instant FromUnixTimeTicks(long ticks)
         public long ToUnixTimeSeconds() =>
             duration.FloorDays * (long) SecondsPerDay + duration.NanosecondOfFloorDay / NanosecondsPerSecond;
 
+        /// <summary>
+        /// Gets the number of seconds since the Unix epoch, along with remaining nanoseconds.
+        /// Negative values represent instants before the Unix epoch.
+        /// </summary>
+        /// <remarks>
+        /// If the number of nanoseconds in this instant is not an exact number of seconds
+        /// the value is truncated towards the start of time, 
+        /// </remarks>
+        /// <value>The number of seconds since the Unix epoch.</value>
+        [Pure]
+        [TestExemption(TestExemptionCategory.ConversionName)]
+        public long ToUnixTimeSecondsAndNanoseconds(out int nanoseconds)
```

Comment:
It seems somewhat weird not to offer an `Instant.FromUnixTimeSecondsAndNanoseconds(seconds, nanoseconds)` to parallel this?

Assuming we really don't want to(?), we should probably document what the code to do that _would_ look like in the doc-comments for this method.

Reply:
(from the conversation on the PR, I assume that'd be `Instant.FromUnixTimeSeconds(seconds).PlusNanoseconds(nanos)`.)

Reply:
Yes, that's exactly right - will add that to the docs.